src/index.js

```diff
@@ -82,11 +71,26 @@ export default function Loadable<Props: {}, Err: Error>(opts: Options) {
       load();
     }
 
-    state = {
-      error: outsideError,
-      pastDelay: false,
-      Component: outsideComponent
-    };
+    constructor(props) {
+      super(props);
+
+      if (isWebpack && webpackRequireWeakId) {
```

Comment:
Could you guard this with another check to see if `outsideComponent` already exists?